src/App.vue

```diff
@@ -0,0 +1,64 @@
+      alt="user avatar"
+      class="header-avatar"
+    />
+  </header>
+
+  <RouterView class="routerView" />
+
+  <footer class="footer">
+    <p>© Copyright <b>2023</b></p>
+
+    <p><strong>Genesis front-end</strong></p>
+  </footer>
```

Comment:
I guess, the same as with footer can be done here. This approach can make the structure of your app more scalable

Reply:
I've answered above